### DISCUSSION TITLE
脑图插件太太太好用啦！推荐

### DISCUSSION THREAD

**Discussion by lillian-who** [obsidianzh/forum#88] [Plugin第三方插件]
先上个图
![image](https://user-images.githubusercontent.com/20553685/121986561-4b470c00-cdc9-11eb-8211-af983dc0af2d.png)

该插件适用于：结构性的markdown文档。虽然不像xmind那么美观，但作为基础脑图使用，来整理思维是很棒的！

**Comment by obsession-obrien** [comment]
怎么搞的，求教程😍😍

**Reply by lillian-who** [reply]
我写了个插件指南的文章，你可以去看[Obsidian新手系列之你不可不知的插件](https://mp.weixin.qq.com/s/XjBHq_xO6PGUWFwzhuHdGQ)

思维导图：Mind Map 章节

**Reply by obsession-obrien** [reply]
Thanks a lot!!

**Comment by ikekwang** [comment]
为啥我的obsidian社区插件永远是空白页

**Reply by Star2227** [reply]
你需要科学上网，
可以在GitHub手动下载，然后放到相应的文件夹

**Comment by guevara** [comment]
可以把百度脑图集成进来

**Reply by wenbopeng** [reply]
本来就支持百度脑图。用irfame就可以了
比如这串代码：
<div style="display: block; position: relative; width: 100%; height: 0px; --aspect-ratio:9/16; padding-bottom: calc(var(--aspect-ratio) * 100%);"><iframe src="https://naotu.baidu.com/home" allow="fullscreen" style="position: absolute; top: 0px; left: 0px; height: 100%; width: 100%;"></iframe></div>


![quicker_549a1531-1e32-497d-9376-b9db06decc9d.png](https://i.loli.net/2021/10/08/Ia7rZjJ1b3tLGuX.png)

**Reply by eightHundreds** [reply]
https://github.com/eightHundreds/obsidian-kity-mind

**Reply by amitbha** [reply]
> https://github.com/eightHundreds/obsidian-kity-mind

空的？什么意思？
